Rakefile

```diff
@@ -0,0 +1,13 @@
+require_relative './config/environment'
+
+puts "Welcome to TicTacToe"
```

Comment:
This was in the Rakefile simply for testing, the bin file had this code for purposes of the actual game but it has now been moved to a CLI class in lib/tic_tac_toe_cli.rb. I've removed the welcome message and commented out the rest of the code in the Rakefile but have kept the file for future reference.